setup.sh

```diff
@@ -81,6 +81,7 @@ EOF
 	echo 'EXTRA_IMAGE_FEATURES += "tools-sdk"' >> conf/local.conf
 	echo 'EXTRA_IMAGE_FEATURES += "tools-debug"' >> conf/local.conf
 	echo 'EXTRA_IMAGE_FEATURES += "package-management"' >> conf/local.conf
+	echo 'DISTRO = "stx"' >> conf/layer.conf
```

Comment:
I see that the distro is renamed to "poky-stx", right? And it should be local.conf instead of layer.conf.